draft-sharabayko-srt-over-quic.md

```diff
@@ -274,13 +275,23 @@ It is also possible for QUIC to not apply any congestion control, relying on SRT
 If the bitrate of the original stream already exceeds network throughput, SRT would still try to deliver it, maintaining congestion and
 eventually breaking the SRT connection.
 
+It is worth mentioning that, in live streaming scenario it may be beneficial to move congestion control mechanisms outside of the protocol towards the encoder (payload producer),
+implementing a network adaptive encoding based on the telemetry provided by SRT and QUIC protocols.
+
 ## Pacing
 
 SRT uses ACK/ACKACK packet pairs to measure RTT on a link, and to track latency and clock drift.
 It also uses packet pair probing to estimate connection bandwidth, although in live configurations
 it has only an informative use.
 
 Buffering and pacing SRT packets by QUIC SHOULD be done with the understanding that would interfere with the corresponding SRT mechanisms.
```

Comment:
```suggestion
Buffering and pacing of SRT packets by QUIC SHOULD be done with the understanding that this would interfere with the corresponding SRT mechanisms.
```